scripts/globals/events/sunbreeze_festival.lua

```diff
@@ -0,0 +1,417 @@
+------------------------------------
+-- Starlight Celebration
+------------------------------------
+require("scripts/globals/settings")
+------------------------------------
+xi = xi or {}
+xi.events = xi.events or {}
+xi.events.sunbreeze_festival = xi.events.sunbreeze_festival or {}
+
+local event = SeasonalEvent:new("SunbreezeFestival")
+
+xi.events.sunbreeze_festival.enabledCheck = function()
+    return tonumber(os.date("%m")) == 8
+end
+
+event:setEnableCheck(xi.events.sunbreeze_festival.enabledCheck)
+
+local sunbreezeNPCEntites =
+{
+    -- Goldfish NPCs
+    17789025, -- Mei (Rabao)
+    17187570, -- Saradorial (West Ronfaure)
+    17216210, -- Fish Eyes (North Gustaberg)
+    17253106, -- Kesha Shopehllok (East Sarutabaruta)
+    -- Bastok Markets (Dancers + Moogle)
```

Comment:
Given that these are in city zones (rabao included) this will all break _immediately_ WHEN (not if) there are client shifts. Highly recommends doing lookups by name and offset.

Reply:
I will have to do this at a later date. I don't have any time to spare before the festival comes up. (Going on vacation in 2 days and I have other obligations to attend to)
But I will put this on my TODO list for when I'm available.